doc/usermanual.md

```diff
@@ -609,78 +606,54 @@ Stores the information about the IoTAgent for further use in the `retrieveVersio
 All contributions to this project are welcome. Developers planning to contribute should follow the [Contribution Guidelines](./docs/contribution.md)
 
 ### Project build
-The project is managed using Grunt Task Runner.
+The project is managed using npm.
 
 For a list of available task, type
 
 ```bash
-grunt --help
+npm run
 ```
 
 The following sections show the available options in detail.
 
 ### Testing
-[Mocha](http://visionmedia.github.io/mocha/) Test Runner + [Chai](http://chaijs.com/) Assertion Library + [Sinon](http://sinonjs.org/) Spies, stubs.
+[Mocha](http://visionmedia.github.io/mocha/) Test Runner + [Should.js](https://shouldjs.github.io/) Assertion Library.
 
-The test environment is preconfigured to run [BDD](http://chaijs.com/api/bdd/) testing style with
-`chai.expect` and `chai.should()` available globally while executing tests, as well as the [Sinon-Chai](http://chaijs.com/plugins/sinon-chai) plugin.
+The test environment is preconfigured to run BDD testing style.
 
 Module mocking during testing can be done with [proxyquire](https://github.com/thlorenz/proxyquire)
 
 To run tests, type
 
 ```bash
-grunt test
-```
-
-Tests reports can be used together with Jenkins to monitor project quality metrics by means of TAP or XUnit plugins.
-To generate TAP report in `report/test/unit_tests.tap`, type
-
-```bash
-grunt test-report
+npm test
 ```
 
 ### Coding guidelines
-jshint, gjslint
+jshint
 
-Uses provided .jshintrc and .gjslintrc flag files. The latter requires Python and its use can be disabled
-while creating the project skeleton with grunt-init.
+Uses provided .jshintrc flag file.
 To check source code style, type
 
 ```bash
-grunt lint
+npm run lint
 ```
 
-Checkstyle reports can be used together with Jenkins to monitor project quality metrics by means of Checkstyle
-and Violations plugins.
-To generate Checkstyle and JSLint reports under `report/lint/`, type
-
-```bash
-grunt lint-report
-```
-
-
 ### Continuous testing
 
 Support for continuous testing by modifying a src file or a test.
 For continuous testing, type
 
 ```bash
-grunt watch
+npm run test:watch
```

Comment:
I have realized that we have "test:watch" and "watch:lib" targets... Is this intentional? I understand that it should be better "test:watch" and "lib:watch" or "watch:test" and "watch:lib" but maybe I'm missing something...